src/imagery/i.eodag/i.eodag.py

```diff
@@ -41,21 +43,21 @@
 
 # %flag
 # % key: e
-# % description: Extract the downloaded the datasets, not considered unless provider is set
+# % description: Extract the downloaded the scenes, not considered unless provider is set
```

Comment:
```suggestion
# % description: Extract the downloaded scenes, not considered unless provider is set
```

Reply:
What does it mean extract in this context and why is it dependent on setting a provider?

Reply:
Scenes are downloaded as ZIP files, and extract here just means whether to unzip these files or not.

It is dependent on the provider because each provider has its own extract configuration that has to be set to either True or False.

Nevertheless, I have found a way to make the `extract`, `delete_archive`, and `output` options independent of the provider. 
Here https://github.com/OSGeo/grass-addons/pull/1136/commits/8870f158b62b9c934f6621b08332af9d8b8183f8